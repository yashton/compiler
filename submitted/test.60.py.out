(KEYWORD if)
(PUNCT "(")
(ID "n")
(PUNCT "==")
(LIT 5)
(PUNCT ")")
(NEWLINE)
(INDENT)
(ID "hello")
(NEWLINE)
(DEDENT)
(ID "there")
(ENDMARKER)
