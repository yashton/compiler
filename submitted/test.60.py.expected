(KEYWORD if)
(PUNCT "(")
(ID "n")
(PUNCT "==")
(LIT 5)
(PUNCT ")")
(NEWLINE)
(INDENT)
(ID "hello")
(NEWLINE)
(DEDENT)
(ERROR "mis-indented program")
